site/viamguides/src/rgbled/rgbled.md

```diff
@@ -0,0 +1,274 @@
+   ![add component](assets/addComponent.png)
+1. Select `board`, and find the `raspberry-pi:rpi4` module. This adds the module for working with the Raspberry Pi 4's GPIO pins. Leave the default name `board-1` for now.
+1. Notice adding this module adds the board hardware component called `board-1`. The collapsible card on the right corresponds to the part listed in the left sidebar.
+   ![added board](assets/addBoard.png)
+1. Click **Save** in the top right to save and apply your configuration changes.
+1. Expand the **TEST** section of the panel to experiment with writing to physical pin `12`. For example, try setting a `High` signal.
+   ![test GPIO pins on board](assets/testBoard.png)
+   > aside negative
+   > Since we are using an RGB LED, you can provide a [PWM (Pulse Width Modulation)](https://en.wikipedia.org/wiki/Pulse-width_modulation) signal to control the brightness and mix colors by adjusting the `duty cycle` and `frequency` of each color channel (Red, Green, and Blue). A constant high signal would result in the LED staying at full brightness without any dimming effect, while PWM allows smooth transitions and precise color blending.
+1. Set the signal to `Low` (for each pin previously set to `High`) to turn off the LED.
+
+> aside negative
+> **TROUBLESHOOTING**: Double check the wiring on your LED component. Also check under the **LOGS** tab to see what might be going wrong.
+
+You can manually and programmatically use the GPIO pins of the [`board`](https://docs.viam.com/components/board/) component to send PWM signals to control your LED. However, to streamline the following steps, let's use a prebuilt module from the Viam registry in the next section.
+
+### Configure your RGB LED
+
+1. In [the Viam app](https://app.viam.com/fleet/locations) under the **CONFIGURE** tab, click the **+** icon in the left-hand menu and select **Component**.
+1. Select `generic`, and find the `led:rgbled` module. This adds the module for controlling your RGB LED. Name the component `rgb-led`.
+   ![select generic rgb led component](assets/addGeneric.png)
+1. In the new `rgb-led` panel, configure your component by adding the following attributes in the **CONFIGURE** field. This tells your LED component to use specific pins on a specific board (called `board-1` in the Viam app).
+   ```json
+   {
+     "board": "board-1",
+     "red_pin": "33",
+     "green_pin": "32",
+     "blue_pin": "12"
+   }
+   ```
+1. Select your board `board-1` from the **Depends on** field.
+   ![configure led](assets/configureLED.png)
+1. Click **Save** to apply your configuration changes. This may take a moment.
+1. Since we are using a generic component, let's test it out under the **CONTROL** tab. Find your `rgb-led` component on this page. Expand the **DO COMMAND** field, input the following code, and hit **Execute**.
+   ```json
+   {
+     "control_rgb_led": {
+       "red": 0.8,
+       "green": 0.5,
+       "blue": 0.2,
+       "duration": 2.0
```

Comment:
What is duration doing here? What kind of result should I see from this command, i.e. a particular color or blinking? If someone mixed up their wires they might see a different color.